src/main/kotlin/no/skatteetaten/aurora/cantus/ApplicationConfig.kt

```diff
@@ -42,9 +42,14 @@ class ApplicationConfig {
         newFixedThreadPoolContext(threadPoolSize, "cantus")
 
     @Bean
-    fun webClient(builder: WebClient.Builder, tcpClient: TcpClient) =
+    fun webClient(
+        builder: WebClient.Builder,
+        tcpClient: TcpClient,
+        @Value("\${spring.application.name}") applicationName: String
+    ) =
         builder
             .defaultHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
+            .defaultHeader("KlientID", applicationName)
```

Comment:
Her burde vi også sette `defaultHeader(AuroraHeaderFilter.KORRELASJONS_ID, RequestKorrelasjon.getId())`

Hvis denne klienten er brukt inne i en coroutine må man i tilegg kopiere noe kode fra Mokey for å få det rett. Denne koden som setter mdc inn i en coroutine context. Spør meg hvis du lurer.

Reply:
denne klienten er brukt både inni en coroutine og uten. Tar en kikk i Mokey.

Reply:
lagt til nå